test/system/my/request_mentor_feedback_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'application_system_test_case'
+
```

Comment:
What do you think about adding a second test for the case where it's the notification shouldn't get added, to guard against a future accidental regression?

Reply:
I will add this to the service level test. Thanks for reminding me.